tests/boost/test_prometheus_metrics.cpp

```diff
@@ -5,6 +5,7 @@
 #include <boost/test/included/unit_test.hpp> // for BOOST_PP_IIF_1, BOOST_CHECK, BOOST_PP_BOOL_2
 #include <iostream>                          // for cout, ostream
 #include <string>                            // for string, allocator, basic_string, operator==
+#include <math.h>                            // for sqrt
```

Comment:
I think this should be `<cmath>`.

Reply:
/code/kotekan/tests/boost/test_prometheus_metrics.cpp:8:10: fatal error: cmath.h: No such file or directory
 #include <cmath.h>                           // for sqrt